### PR TITLE
Validation added for project duration 

### PR DESCRIPTION
**Pull request by SrinivasaDontiboyina**
### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

### CODE REVIEW

**Review by tommyb82 — APPROVED** [review]
Great, just minor formatting / style comments 👍

File: src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ValidationService.java [hunk 1]
```diff
@@ -34,6 +37,8 @@ public class ValidationService {
   private static final Integer AWARD_STATUS = 500;
   private static final Integer ABANDONED_STATUS = 1500;
 
+  private static final Period FOUR_YEAR_PERIOD=java.time.Period.parse("P4Y");
```
[the remark this answers is not in the document]
Comment (SrinivasaDontiboyina) [reply src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ValidationService.java:40]: done

File: src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ValidationService.java [hunk 2]
```diff
@@ -169,6 +174,29 @@ public void validateEventTypeBeforeUpdate(final ExportRfxResponse exportRfxRespo
     }
   }
 
+  public void validateProjectDuration(List<QuestionNonOCDSOptions> optionList) {
+
+    if (!CollectionUtils.isEmpty(optionList) && optionList.size() == 1) {
+      QuestionNonOCDSOptions projectDurationOptionValue = optionList.get(0);
+      try {
+        Period projectDuration = Period.parse(projectDurationOptionValue.getValue());
+        LocalDate now = LocalDate.now();
```
[the remark this answers is not in the document]
Comment (SrinivasaDontiboyina) [reply src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ValidationService.java:183]: done

**Comment by SrinivasaDontiboyina** [comment]
Review comments incorporated and merged with develop.

**Pull request merged**